histogram/src/atomic.rs

```diff
@@ -54,23 +51,18 @@ impl AtomicHistogram {
         Ok(())
     }
 
-    /// Produce a snapshot from this histogram.
-    pub fn snapshot(&self) -> Snapshot {
-        let end = SystemTime::now();
-
+    /// Read the bucket values into a new `Histogram`
+    pub fn load(&self) -> Histogram {
```

Comment:
Not a strong opinion, but is `load` the best term for this? Seems more like a `store` to me...

Reply:
This is a load. We're loading the value(s) from the atomics and returning the non-atomic type. This mirrors https://doc.rust-lang.org/std/sync/atomic/struct.AtomicU64.html#method.load but we do not take an ordering.

Reply:
Fair enough